### PR TITLE
Various improvements to config handling, fixes #6

### LINKED ISSUE #6: also support tox.ini for config
[flake8 allows storing the config in `tox.ini`](http://flake8.pycqa.org/en/latest/user/configuration.html). I like this because it keeps the 'task' and 'task config' close together ([example here](https://github.com/jieter/django-tables2/blob/f30fbb95d6c95fbe05677a10393c5d43a3e29724/tox.ini#L50-L58)). Is this something you would consider adding to potypo?

### PR DESCRIPTION
**Pull request by jieter**
This PR:

 - also looks in `tox.ini` for config (fixes #6)
 - allows `filters` and/or `chunkers` to be absent from the config.
 - allows spaces in items of `filters` and `chunkers`, preventing errors like these:
```
Traceback (most recent call last):
  File "/Users/jieter/.virtualenvs/django-tables2/bin/potypo", line 11, in <module>
    sys.exit(main())
  File "/Users/jieter/.virtualenvs/django-tables2/lib/python3.6/site-packages/potypo/__main__.py", line 31, in main
    mod = __import__(components[0], fromlist=[components[1]])
ModuleNotFoundError: No module named ' enchant'
```
- factors out the logic to load modules from the config into a function.

### CODE REVIEW

File: potypo/__main__.py [hunk 1]
```diff
@@ -8,32 +8,27 @@
...
-def main():
-    config = configparser.ConfigParser()
-    config.read('setup.cfg')
-    conf = config['potypo']
 
-    chunker_list = []
-    for chunker in conf['chunkers'].strip().split(","):
-        if "." in chunker:
-            components = chunker.rsplit('.',1)
+def load_classes(module, config):
```
Comment (jieter) [inline potypo/__main__.py:12]: This method probably needs a better name, but I couldn't think of any.

File: potypo/__main__.py [hunk 2]
```diff
@@ -8,32 +8,27 @@
...
+        if '.' in item:
+            components = item.rsplit('.', 1)
             mod = __import__(components[0], fromlist=[components[1]])
             class_object = getattr(mod, components[1])
         else:
-            class_object = getattr(chunkers, chunker)
+            class_object = getattr(module, item)
 
-        chunker_list.append(class_object)
+        yield class_object
```
Comment (koebi) [inline potypo/__main__.py]: This should return a list and not yield the objects, as just discussed :)

**Review by koebi — CHANGES_REQUESTED** [review]

File: potypo/__main__.py (hunk 1 above)
Comment (koebi) [reply potypo/__main__.py:12]: Works for me :)

File: potypo/__main__.py [hunk 3]
```diff
@@ -8,32 +8,29 @@
...
-            components = f.rsplit('.',1)
-            mod = __import__(components[0], fromlist=[components[1]])
-            class_object = getattr(mod, components[1])
-        else:
-            class_object = getattr(filters, f)
 
-        filter_list.append(class_object)
+def main():
+    config = configparser.ConfigParser()
+    config.read(['setup.cfg', 'tox.ini'])
```
Comment (koebi) [inline potypo/__main__.py:29]: This is super neat and a great find :)

**Review by koebi — APPROVED** [review]
Thank you very much :)

**Comment by jieter** [comment]
yay!

**Pull request merged**